src/index.html

```diff
@@ -11,7 +11,186 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
+    <form
+    action="https://mate-academy-form-lesson.herokuapp.com/create-application"
+    method="post"
+    autocomplete="off"
+    >
+      <fieldset class="fieldset">
+        <legend>Personal information</legend>
+
+        <div class="form-field">
+          <label for="surname">Surname:</label>
+          <input
+            type="text"
+            name="surname"
+            id="surname"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="name">Name:</label>
+          <input
+            type="text"
+            name="Name"
+            id="name"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="age">How old are You?</label>
+          <input
+            type="number"
+            name="Age"
+            id="age"
+            value="12"
+            min="1"
+            max="100"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="date-of-birth">Full date of birth:</label>
+          <input
+            type="date"
+            name="date-of-birth"
+            id="date-of-birth"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="terms-and-conditions">I accept the term of agreement</label>
+          <input
+            type="checkbox"
+            name="terms-and-conditions"
+            id="terms-and-conditions"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <div class="form-field">
+          <label for="email">E-mail:</label>
+          <input
+            type="email"
+            name="email"
+            id="email"
+            placeholder="email@example.com"
+            required
+          >
+        </div>
+
+        <div class="form-field">
+          <label for="password">Password:</label>
+          <input
+            type="password"
+            name="password"
+            id="password"
+            minlength="5"
+            maxlength="32"
+            required
+          >
+        </div>
+      </fieldset>
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
```

Comment:
In one place you have spaces between html elements in another you don't have. Keep one style, remove spaces